website/blog/2020-08-04-prestodb-and-hudi.md

```diff
@@ -0,0 +1,153 @@
+- **Access to fresh data** :  It's not everyday we will find that reduced resource usage also results in improved performance, since typically we add more resources (e.g memory) to improve performance metric (e.g query latency) . By fundamentally shifting away from how datasets have been traditionally managed for may be the first time since the dawn of the big data era, Hudi, in fact, realizes this rare combination. A sweet side-effect of incrementalizing batch-processing is that the pipelines also take a much smaller amount of time to run, putting data into hands of organizations much more quickly, than it has been possible with data-lakes before.
+- **Unified Storage** : Building upon all the three benefits above, faster and lighter processing right on top of existing data-lakes mean lesser need for specialized storage or data-marts, simply for purposes of gaining access to near real-time data.
+
+## Types of Hudi tables and queries
+
+### Table Types
+Hudi supports the following table types.
+**Copy On Write (COW)** : Stores data using exclusively columnar file formats (e.g parquet). Updates version & rewrite the files by performing a synchronous merge during write.
+**Merge On Read (MOR)** : Stores data using file versions with combination of columnar (e.g parquet) + row based (e.g avro) file formats. Updates are logged to delta files & later compacted to produce new versions of columnar files synchronously or asynchronously.
+
+Following table summarizes the trade-offs between these two table types.
+
+
+| Trade-off           | CopyOnWrite                     | MergeOnRead     |
+| ---------           | -----------                     | -----------     |
+| Data Latency        | Higher                          | Lower           |
+| Update cost (I/O)   | Higher (rewrite entire parquet) | Lower (append to delta log) |
+| Parquet File Size   | Smaller (high update(I/0) cost) | Larger (low update cost) |
+| Write Amplification | Higher                          | Lower (depending on compaction strategy) |
+
+### Query types
+Hudi supports the following query types.
+
+***Snapshot Queries*** : Queries see the latest snapshot of the table as of a given commit or compaction action. In case of merge-on-read table, it exposes near-real time data(few mins) by merging the base and delta files of the latest file version on-the-fly. For copy-on- write tables, it provides a drop-in replacement for existing parquet tables, while providing upsert/delete and other write side features.
```

Comment:
Remove the space before the colons. Add a space before the open parentheses.

Remove a space after the hyphen in copy-on-